Source/XUnitExtensions/Lit/LitTestConfiguration.cs

```diff
@@ -1,22 +1,20 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Reflection;
-using Xunit.Abstractions;
 
 namespace XUnitExtensions.Lit {
 
   // TODO: Make safely immutable
   public class LitTestConfiguration {
-    public readonly Dictionary<string, string> Substitutions;
+    public readonly Dictionary<string, object> Substitutions;
```

Comment:
This looks like a useful change, but can you document what actual types are supported as substitution values? AFAICT it's either `string` or  `IEnumerable<string>`.